src/__swaps__/screens/Swap/components/TokenList/TokenToBuyList.tsx

```diff
@@ -180,7 +185,15 @@ export const TokenToBuyList = () => {
             />
           );
         }}
-        renderScrollComponent={ScrollViewWithRef as ComponentType<ScrollViewProps>}
+        renderScrollComponent={props => {
+          return (
+            <Animated.ScrollView
+              // eslint-disable-next-line react/jsx-props-no-spreading
+              {...props}
+              animatedProps={animatedListProps}
+            />
+          );
+        }}
```

Comment:
just make sure this doesn't bring back that bug with favoriting and not rerendering correctly

Reply:
Haven't seen any issues, thinking it must have been a FlashList issue